packages/cli/src/scaling/pubsub/pubsub-handler.ts

```diff
@@ -156,7 +157,7 @@ export class PubSubHandler {
 		'relay-execution-lifecycle-event': async ({ type, args, pushRef }) => {
 			if (!this.push.getBackend().hasPushRef(pushRef)) return;
 
-			this.push.send(type, args, pushRef);
+			this.push.send({ type, data: args } as PushMessage, pushRef);
```

Comment:
instead of casting `as PushMessage` here, maybe we could update the types for this event in `packages/cli/src/events/maps/pub-sub.event-map.ts`?

Reply:
Ended up removing all generics from push as well: [75b9e71](https://github.com/n8n-io/n8n/pull/12331/commits/75b9e71ce4241fa88834d79439fdafab9cd551b9)